EditorExtensions/Resources/server/services/srv-scss.js

```diff
@@ -35,8 +35,9 @@ var handleSass = function(writer, params) {
             writer.end();
         }
 
-        var css = result.css.toString();
-        var map = result.map.toString();
+        var map, css = result.css.toString();
+        if (!options.omitSourceMapUrl)
+            map = result.map.toString();
```

Comment:
It might be more clear if I had wrote it like this :
```C#
var map, css = result.css.toString();
if (params.sourceMapURL !== undefined)
  map = result.map.toString();
```